code/modules/unit_tests/reagent_transfer.dm

```diff
@@ -0,0 +1,32 @@
+/// Tests transferring reagents between two reagent_containers, making sure all the regent transfers
+/// leaving nothing behind in the source container and making sure everything is in the target container.
+/datum/unit_test/reagent_transfer
+
+/datum/unit_test/reagent_transfer/Run()
+	var/obj/item/reagent_containers/cup/glass/bottle/source_container = allocate(/obj/item/reagent_containers/cup/glass/bottle)
+	var/obj/item/reagent_containers/cup/glass/bottle/target_container = allocate(/obj/item/reagent_containers/cup/glass/bottle)
```

Comment:
Can you instead create the `/datum/reagents` directly? This and the clearing code looks like overkill.

Reply:
Done.